templates/CRM/ACL/Form/WordPress/Permissions.tpl

```diff
@@ -12,42 +12,44 @@
   <p>{ts}Use this form to Grant access to CiviCRM components and other CiviCRM permissions to WordPress roles.{/ts}</p>
   <p>{ts}<strong>NOTE: Super Admin</strong> and <strong>Administrator</strong> roles will have all permissions in CiviCRM.{/ts}</p>
 </div>
-<div class="crm-block crm-form-block crm-export-form-block">
+<div class="crm-block crm-form-block crm-export-form-block crm-sticky">
   <table>
-    <tr>
-      <td class="label">&nbsp;</td>
-      {assign var="num" value=0}
-      {foreach from=$roles key=role_name item=role_value}
-        <td align="center"><strong>{$role_value}</strong></td>
-        {assign var="num" value=$num+1}
-      {/foreach}
-    </tr>
-
-    {assign var="x" value=0}
-    {foreach from=$table key=perm_name item=row}
-      {if $x mod 2 eq 1}
-        <tr style="background-color: #E6E6DC;">
-      {else}
-        <tr style="background-color: #FFFFFF;">
-      {/if}
-
-      <td style="height: 2.6em;">
-        {$row.label}
-        {if !empty($row.desc)}
-          <br/><span class="description">{$row.desc}</span>
+    <thead>
+      <tr>
+        <th class="label">&nbsp;</th>
+        {assign var="num" value=0}
+        {foreach from=$roles key=role_name item=role_value}
+          <th align="center"><strong>{$role_value}</strong></th>
+          {assign var="num" value=$num+1}
+        {/foreach}
+      </tr>
+    </thead>
+    <tbody>
+      {assign var="x" value=0}
+      {foreach from=$table key=perm_name item=row}
+        {if $x mod 2 eq 1}
+          <tr style="background-color: #E6E6DC;">
+        {else}
+          <tr style="background-color: #FFFFFF;">
         {/if}
-      </td>
 
-      {foreach from=$row.roles key=index item=role_name}
-        <td align="center" style="padding-top: 0.6em;">
-          {$form.$role_name.$perm_name.html}
+       <td style="height: 2.6em;">
+           {$row.label}
+          {if !empty($row.desc)}
+            <br/><span class="description">{$row.desc}</span>
+          {/if}
         </td>
-      {/foreach}
 
-      </tr>
-      {assign var="x" value=$x+1}
-    {/foreach}
+        {foreach from=$row.roles key=index item=role_name}
+          <td align="center" style="padding-top: 0.6em;">
```

Comment:
Just noticed this inline styles here and at line 36 - can this be avoided?

Reply:
I think that was there before but I agree it's not ideal really.

Reply:
Probably shouldn't block the PR over this since it's pre-existing.

Reply:
I'm actually going to create a bigger WIP pertaining to all the tables and stuff but I just wanted to put this through so that I could tackle that after.